DESCRIPTION

```diff
@@ -26,7 +26,8 @@ Description: Download and import of 'OpenStreetMap' ('OSM') data as 'sf'
     routines for return to 'R'.
 License: GPL-3
 URL: https://docs.ropensci.org/osmdata/ (website)
-    https://github.com/ropensci/osmdata/ (devel)
+    https://github.com/ropensci/osmdata/ (devel),
+    https://github.com/ropensci/osmdata
```

Comment:
Is this needed? Seems redundant.

Reply:
Yeah, i agree that it's redundant. It's my first time actually trying all the things auto-generated in #365, and that one was generated by `usethis::use_github_links()`. Feel free to delete anytime.

Reply:
I'll have a look because it ends up generating a broken link. https://github.com/ropensci/osmdata/blob/e032ee8e70e4dd3ff34b63986c3d0b7bdb5b0594/man/osmdata.Rd#L110

Reply:
Fixed in https://github.com/ropensci/osmdata/commit/86c22e9005187fefd96c5c9258b3f05213a22985